### DISCUSSION TITLE
Popout Webste on zen browser

### DISCUSSION THREAD

**Discussion by BraveBrowserFan** [zen-browser/desktop#3931] [Ideas]
Hi

I like to see websites that when I click the url it will popout like this on arc browser thanks

![image](https://github.com/user-attachments/assets/145d10a9-3cd9-49b4-bf5c-a0a7d796f7be)

**Comment by 12th-devs** [comment]
This is called Zen Glance in settings, it is alt + click or ctrl + click, you can change it in settings :}
